docs/src/main/sphinx/admin/properties-client-protocol.md

```diff
@@ -79,36 +80,49 @@ Threshold for enabling compression with larger segments.
 ### `protocol.spooling.initial-segment-size`
 
 - **Type:** [](prop-type-data-size)
-- **Default value:** 8MB
+- **Default value:** `8MB`
+- **Minimum value:** `1KB`
+- **Maximum value:** `128MB`
+- **Session property:** `spooling_initial_segment_size`
 
 Initial size of the spooled segments.
 
 ### `protocol.spooling.maximum-segment-size`
 
 - **Type:** [](prop-type-data-size)
-- **Default value:** 16MB
+- **Default value:** `16MB`
+- **Minimum value:** `1KB`
+- **Maximum value:** `128MB`
+- **Session property:** `spooling_max_segment_size`
```

Comment:
Let me rename this session property to maximum for consistency

Reply:
Updated